### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by excllent123** [excllent123/My_Kaggle_Solution#1]
model = nn.Linear(10, 2)
   optimizer = optim.SGD(model.parameters(), lr=1.)
   steps = 10
   scheduler = optim.lr_scheduler.CosineAnnealingLR(optimizer, steps)

   for epoch in range(5):
       for idx in range(steps):
           scheduler.step()
           print(scheduler.get_lr())
    
    print('Reset scheduler')
    scheduler = optim.lr_scheduler.CosineAnnealingLR(optimizer, steps)

**Comment by excllent123** [comment]
Done
